app/worker.py

```diff
@@ -0,0 +1,39 @@
+from importlib import import_module
+
+from .application import db
+from .settings.arq import settings
+from .settings.globals import DATABASE_CONFIG, ARQ_BACKGROUND_FUNCTIONS
+
+
+FUNCTIONS: list = [
+    getattr(import_module(function_path), function_name)
```

Comment:
pydantic [`import_string`](https://github.com/samuelcolvin/pydantic/blob/master/pydantic/utils.py#L36) is for exactly this purpose.

Reply:
Did not know this existed - thank you for the heads up here